src/wmnds/components/form-elements/file-upload/_file-upload.njk

```diff
@@ -0,0 +1,64 @@
+{% macro wmndsFileUpload(params) %}
+
+{# Imports #}
+{% from "wmnds/components/icon/_icon.njk" import wmndsIcon %}
+
+<div class="wmnds-fe-group">
+  <fieldset class="wmnds-fe-fieldset">
+    <legend class="wmnds-fe-fieldset__legend">
+      <h3 class="wmnds-fe-question">
+        Upload a file
+      </h3>
+      <p>
+        Files must be jpeg or png file format and be less than 2mb
+      </p>
+    </legend>
+    {% if params.fileSelected -%}
+      <div class="wmnds-fe-file-upload">
+        <input
+          type="file"
+          name="fileUploaded"
+          id="fileUploaded"
+          class="wmnds-fe-file-upload__input wmnds-fe-file-upload__input--file-selected"
+        />
+        <label for="fileUploaded" class="wmnds-btn wmnds-btn--primary wmnds-fe-file-upload__label">
+          Remove file
+          {#- Get SVG icon -#}
+          {{
+            wmndsIcon({
+              icon: 'general-trash',
+              class: 'wmnds-btn__icon wmnds-btn__icon--right'
+            }) | indent(4) | trim
+          }}
+        </label>
+        <span>
+          test-file.jpg
+        </span>
+      </div>
+    {%- else -%}
+      <div class="wmnds-fe-file-upload">
+        <input
+          type="file"
+          name="fileUpload"
+          id="fileUpload"
+          class="wmnds-fe-file-upload__input"
+        />
+        <label for="fileUpload" class="wmnds-btn wmnds-btn--primary wmnds-fe-file-upload__label">
+          Choose file
+          {#- Get SVG icon -#}
+          {{
+            wmndsIcon({
+              icon: 'general-paperclip',
+              class: 'wmnds-btn__icon wmnds-btn__icon--right'
+            }) | indent(4) | trim
+          }}
+        </label>
```

Comment:
The indentation of the svg tag might be a bit confusing.

![image](https://user-images.githubusercontent.com/43111519/90727925-f4ad9600-e2bb-11ea-911b-40b3cd6b6586.png)